am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/GrantAccessIntegrationTest.java

```diff
@@ -75,4 +77,29 @@ void whenCreatingDuplicateResourceAccessEntryIsOverwritten() {
 
         assertThat(databaseHelper.countExplicitPermissions(resourceId)).isEqualTo(1);
     }
+
+    @Test
+    void whenGrantingAccessForMultipleUsersEntriesShouldAppearInDatabase() {
+        Map<JsonPointer, Set<Permission>> multipleAttributePermissions = new ConcurrentHashMap<>();
```

Comment:
No more ConcurrentHashMap please. We have Guava in test scope.

Reply:
Why not to use one of the methods that grant access to whole doc? Does it matter for this particular test what attribute access is granted for?

Reply:
👍